clang/lib/DPCT/DPCT.cpp

```diff
@@ -736,8 +739,33 @@ int runDPCT(int argc, const char **argv) {
   CudaPath = getCudaInstallPath(OriginalArgc, argv);
   DpctDiags() << "Cuda Include Path found: " << CudaPath << "\n";
 
-  RefactoringTool Tool(OptParser->getCompilations(),
-                       OptParser->getSourcePathList());
+  std::vector<std::string> SourcePathList;
+  if (QueryAPIMapping.getNumOccurrences()) {
+    APIMapping::initEntryMap();
+    auto SourceCode = APIMapping::getAPISourceCode(QueryAPIMapping);
+    if (SourceCode.empty()) {
+      llvm::outs() << "The API Mapping is not available\n";
+      dpctExit(MigrationSucceeded);
```

Comment:
let's define 2 error code for API mapping query:  1. API mapping not exist.  2. internal error (tmp file create fail)